addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -233,6 +271,8 @@ var WysiwygMultizone = Wysiwyg.extend({
      * @returns {$.Promise} resolve with true if the content was dirty
      */
     save: function () {
+        // Restore dropdown behavior
+        this.$el.find('.dropdown-toggle-megamenu').off().attr('data-toggle', 'dropdown').dropdown('hide');
```

Comment:
- `this.$el.find(` -> `this.$(`
- Instead of doing `.on('click', ....)` do `.on('click.name_of_the_widget', ...)` then here you can do `.off('.name_of_the_widget')` so that you only remove the related event handlers. This is jQuery namespaces.